command/command.go

```diff
@@ -7,6 +7,8 @@ import (
 	"regexp"
 	"time"
 
+	"github.com/asaskevich/govalidator"
+	"github.com/pkg/errors"
```

Comment:
These imports were below and with a new line from the Iocal ones to separate them (it's idiomatic doing this with go just for clearness). Can you please remove this change?